__tests__/endpoints.test.js

```diff
@@ -320,4 +320,32 @@ describe('NC news endpoint tests',()=>{
             })
         })
     })
+    describe('DELETEting news',()=>{
+        describe('DELETE the comment by comment id',()=>{
+            test('Respond with 204 and no content', ()=>{
+                return request(app)
+                .delete('/api/comments/1')
+                .expect(204)
```

Comment:
You can probably just test to this point, when you send a 204 no content will be sent automatically